packages/graph/src/graph2/graph.ts

```diff
@@ -1241,6 +1250,7 @@ Graph2.prototype.publish("tooltipWidth", 256, "number", "Tooltip width (pixels)"
 Graph2.prototype.publish("tooltipHeight", 128, "number", "Tooltip width (pixels)");
 Graph2.prototype.publish("enableTooltipPointerEvents", false, "boolean", "If true, tooltip will use the style: 'pointer-events: all'");
 Graph2.prototype.publish("tooltipCloseDelay", 0, "number", "Number of milliseconds to wait before closing tooltip (cancelled on tooltip mouseover event)");
+Graph2.prototype.publish("doubleClickMaxDelay", 0, "number", "Number of milliseconds to wait before a subsequent click is not considered a double click");
```

Comment:
@GordonSmith the vertex_dblclick was working after your recent fix so I set this to 0 rather than 500. Thinking it could be used as a fall back